tests/nosetests/pyanaconda_tests/modules/boss/modules_test.py

```diff
@@ -64,7 +64,7 @@ def start_one_module_test(self, gio):
         ]
 
         task = StartModulesTask(self._message_bus, service_names, addons_enabled=False)
-        (observer, ) = self._check_started_modules(task, service_names)
+        (observer, ) = self._check_started_modules(task, service_names)  # pylint: disable=unbalanced-tuple-unpacking
```

Comment:
This is weird. I think it should be `observers = ` based on other usages of the method.